backend/siarnaq/api/episodes/models.py

```diff
@@ -338,3 +425,89 @@ class Meta:
 
     def __str__(self):
         return f"{self.tournament} ({self.name})"
+
+    def enqueue(self):
+        """Creates and enqueues all matches for this round.
+        Fails if this round is already in progress."""
+
+        tournament = json.loads(
+            challonge.get_tournament(self.tournament.challonge_id_private)
+        )
+        # Derive matches of this round
+        matches = []
+        # kes some wrangling with API response format
+        for item in tournament["included"]:
+            if item["type"] == "match":
+                round_idx = item["attributes"]["round"]
+                if round_idx == self.challonge_id:
+                    # Only enqueue the round if all matches are ready and open.
+                    # TODO create a force-requeue,
```

Comment:
Force-requeue is available on the Matches page. I think it's unnecessary for the tournament / round.

Reply:
wait to clarify -- this is force-requeue all matches in round, if round completely dies

Reply:
oh wait shoot massive typo lolol